### ISSUE TITLE
Dual rate menu also includes  the Exponential  menu and should be renamed to reflect this

### ISSUE THREAD

**Issue by prondel** [FrSky-OS/discussion#19]
Dual rate menu  is also EXP menu and should be renamed to reflect this

![image](https://cloud.githubusercontent.com/assets/16406160/11965963/be5763a8-a8f8-11e5-9b26-0dcfcd5714f0.png)

**Comment by MIKE-DAILEY** [comment]
Can you explain how you created a dual rate on this screen?  I want to adjust rate by percentage.

**Comment by prondel** [comment]
By using one of the D/R curves, and then edit the curve on the right :(

**Comment by prondel** [comment]
I agree this is not convenient at all. For D/R we should have the possibility to set a % as you said.

**Comment by MIKE-DAILEY** [comment]
That's what I thought, but there is only D/R 1 and D/R 2 and no D/R 3?  Seems like they have Expo and rates mixed up a bit.

**Comment by prondel** [comment]
Yes Mike, D/R and EXP are combined :( !
On my side, I can see up to D/R 5 curve

**Comment by MIKE-DAILEY** [comment]
OK, thanks I see 1 - 5 now, but have you figured out how are we getting the rate set to a switch position?

**Comment by prondel** [comment]
Same for me. We should  be able to select a switch. This point another problem. As D/R and EXP are combined, we cannot activate one and not the other :(

**Comment by AloftHobbies** [comment]
I think it is fair to say that we would all like to see a large change to this feature. While technically this might be able to get the job done, it really needs to be done differently. Again, I think Spektrum has done a very nice job for dual rates and expo, here is a short video sample.
https://www.youtube.com/watch?v=5sdrq13aJ-4
